I've erred on the side of being very verbose since the user should be aware of what's happening:

```
2018-02-05 11:56:46,540 2405       [      main] INFO       ome.formats.OMEROMetadataStoreClient - Insecure connection requested, falling back
2018-02-05 11:56:46,749 2614       [      main] WARN       ome.formats.OMEROMetadataStoreClient - Cannot fallback. Using secure connection
Ice.ConnectionRefusedException: java.net.ConnectException: Connection refused
	at IceInternal.ConnectRequestHandler.getConnection(ConnectRequestHandler.java:244) ~[ice.jar:na]
	at IceInternal.ConnectRequestHandler.sendRequest(ConnectRequestHandler.java:141) ~[ice.jar:na]
	at IceInternal.Outgoing.invoke(Outgoing.java:77) ~[ice.jar:na]
	at Ice._ObjectDelM.ice_isA(_ObjectDelM.java:33) ~[ice.jar:na]
	at Ice.ObjectPrxHelperBase.ice_isA(ObjectPrxHelperBase.java:98) ~[ice.jar:na]
	at Ice.ObjectPrxHelperBase.ice_isA(ObjectPrxHelperBase.java:61) ~[ice.jar:na]
	at Glacier2.RouterPrxHelper.checkedCast(RouterPrxHelper.java:2262) ~[glacier2.jar:na]
	at omero.client.getRouter(client.java:802) ~[blitz.jar:na]
	at omero.client.createSession(client.java:723) ~[blitz.jar:na]
	at omero.client.joinSession(client.java:658) ~[blitz.jar:na]
	at omero.client.createClient(client.java:509) ~[blitz.jar:na]
	at ome.formats.OMEROMetadataStoreClient.unsecure(OMEROMetadataStoreClient.java:745) ~[blitz.jar:na]
	at ome.formats.OMEROMetadataStoreClient.initialize(OMEROMetadataStoreClient.java:705) ~[blitz.jar:na]
	at ome.formats.importer.ImportConfig.createStore(ImportConfig.java:375) [blitz.jar:na]
	at ome.formats.importer.cli.CommandLineImporter.<init>(CommandLineImporter.java:169) [blitz.jar:na]
	at ome.formats.importer.cli.CommandLineImporter.main(CommandLineImporter.java:964) [blitz.jar:na]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method) ~[na:1.8.0_25]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:716) ~[na:1.8.0_25]
	at IceInternal.Network.doFinishConnect(Network.java:367) ~[ice.jar:na]
	at IceInternal.TcpTransceiver.initialize(TcpTransceiver.java:33) ~[ice.jar:na]
	at Ice.ConnectionI.initialize(ConnectionI.java:1933) ~[ice.jar:na]
	at Ice.ConnectionI.message(ConnectionI.java:1084) ~[ice.jar:na]
	at IceInternal.ThreadPool.run(ThreadPool.java:321) ~[ice.jar:na]
	at IceInternal.ThreadPool.access$300(ThreadPool.java:12) ~[ice.jar:na]
	at IceInternal.ThreadPool$EventHandlerThread.run(ThreadPool.java:693) ~[ice.jar:na]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_25]
```

--- a/components/blitz/src/ome/formats/OMEROMetadataStoreClient.java
+++ b/components/blitz/src/ome/formats/OMEROMetadataStoreClient.java
@@ -698,10 +698,17 @@ public class OMEROMetadataStoreClient
     {
         secure(server, port);
         serviceFactory = c.joinSession(sessionKey);
-    if (!isSecure)
-    {
-            unsecure();
-    }
+        if (!isSecure)
+        {
+            try
+            {
+                unsecure();
+            } catch (Ice.ConnectionRefusedException cre)
+            {
+                log.warn("Cannot fallback. Using secure connection", cre);
+            }
+        }
+
         initializeServices(true);
     }
 

--- a/components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java
+++ b/components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java
@@ -608,6 +608,9 @@ public class CommandLineImporter {
         LongOpt outputFormat =
                 new LongOpt("output", LongOpt.REQUIRED_ARGUMENT, null, 25);
 
+        LongOpt encryptedConnection =
+                new LongOpt("encrypted", LongOpt.REQUIRED_ARGUMENT, null, 26);
+
         // DEPRECATED OPTIONS
         LongOpt minutesWaitDeprecated =
                 new LongOpt("minutes_wait", LongOpt.REQUIRED_ARGUMENT, null, 86);
@@ -645,7 +648,7 @@ public class CommandLineImporter {
                                 closeCompleted, waitCompleted, autoClose,
                                 exclude, target, noStatsInfo,
                                 noUpgradeCheck, qaBaseURL,
-                                outputFormat,
+                                outputFormat, encryptedConnection,
                                 plateName, plateName2,
                                 plateDescription, plateDescription2,
                                 noThumbnailsDeprecated,
@@ -802,6 +805,12 @@ public class CommandLineImporter {
                 String outputArg = g.getOptarg();
                 log.info("Setting output format: {}", outputArg);
                 outputChoice = ImportOutput.valueOf(outputArg);
+                break;
+            }
+            case 26: {
+                String encrytpedArg = g.getOptarg();
+                log.info("Setting encrypted: {}", encrytpedArg);
+                config.encryptedConnection.set(Boolean.valueOf(encrytpedArg));
                 break;
             }
             // ADVANCED END ---------------------------------------------------

--- a/components/tools/OmeroPy/src/omero/plugins/import.py
+++ b/components/tools/OmeroPy/src/omero/plugins/import.py
@@ -420,6 +420,11 @@ class ImportControl(BaseControl):
             "--output", choices=OUTPUT_CHOICES,
             help="Set an alternative output style",
             metavar="TYPE")
+        add_java_argument(
+            "--encrypted",
+            choices=("true", "false"),
+            help="Whether the import should use SSL or not",
+            metavar="TYPE")
 
         # Arguments previously *following" `--`
         advjava_group = parser.add_argument_group(

--- a/components/tools/OmeroPy/test/integration/clitest/test_import.py
+++ b/components/tools/OmeroPy/test/integration/clitest/test_import.py
@@ -1175,3 +1175,17 @@ class TestImport(CLITest):
 
         with pytest.raises(NonZeroReturnCode):
             self.do_import(capfd)
+
+    @pytest.mark.parametrize("arg",
+                             ['', '--encrypted=false', '--encrypted=true'])
+    def testEncryption(self, tmpdir, capfd, arg):
+        """Test encryption import"""
+
+        fakefile = tmpdir.join("test.fake")
+        fakefile.write('')
+
+        self.args += [str(fakefile)]
+        self.args += [arg]
+
+        out, err = self.do_import(capfd)
+        assert self.get_object(out, 'Image')

